infra/modules/azure_function_app/examples/complete/locals.tf

```diff
@@ -0,0 +1,26 @@
+locals {
+  environment = {
+    prefix          = "io"
+    env_short       = "p"
+    location        = "italynorth"
+    domain          = "modules"
+    app_name        = "test"
+    instance_number = "01"
+  }
+
+  location_map = {
```

Comment:
can't we take `location_map` from the naming convention module?

Reply:
we can and we do it 😆